Gerrit tests fails because change was modified

```
_______________________________ test_build_info ________________________________
    def test_build_info():
        ga = gerrit.API("gerrit.ovirt.org")
        res = ga.build_info("96774")
    
>       assert res == {
            "ref": "refs/changes/74/96774/7",
            "url": "git://gerrit.ovirt.org/vdsm",
        }
E       AssertionError: assert {'ref': 'refs...irt.org/vdsm'} == {'ref': 'refs/...irt.org/vdsm'}
E         Common items:
E         {'url': u'git://gerrit.ovirt.org/vdsm'}
E         Differing items:
E         {'ref': 'refs/changes/74/96774/8'} != {'ref': 'refs/changes/74/96774/7'}
E         Full diff:
E         - {'ref': u'refs/changes/74/96774/8', 'url': u'git://gerrit.ovirt.org/vdsm'}
E         ?         -                       ^          -
E         + {'ref': 'refs/changes/74/96774/7', 'url': 'git://gerrit.ovirt.org/vdsm'}
E         ?                                ^
test/gerrit_test.py:8: AssertionError
```

Need to use a merged patch for this test.

--- a/test/gerrit_test.py
+++ b/test/gerrit_test.py
@@ -3,9 +3,9 @@ from oci import gerrit
 
 def test_build_info():
     ga = gerrit.API("gerrit.ovirt.org")
-    res = ga.build_info("96774")
+    res = ga.build_info("98719")
 
     assert res == {
-        "ref": "refs/changes/74/96774/7",
+        "ref": "refs/changes/19/98719/2",
         "url": "git://gerrit.ovirt.org/vdsm",
     }

